crates/bevy_solari/src/realtime/prepare.rs

```diff
@@ -0,0 +1,63 @@
+use super::SolariLighting;
+use bevy_ecs::{
+    component::Component,
+    entity::Entity,
+    query::With,
+    system::{Commands, Query, Res},
+};
+use bevy_math::UVec2;
+use bevy_render::{
+    camera::ExtractedCamera,
+    render_resource::{Buffer, BufferDescriptor, BufferUsages},
+    renderer::RenderDevice,
+};
+
+const RESERVOIR_STRUCT_SIZE: u64 = 32;
```

Comment:
I think there should be a comment in the resevoir shader that this const must be adjusted when the memory layout is changed.
Perhaps this const should also have a comment pointing back to the shader.

Reply:
Added.